### PR TITLE
move-iter-function

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

This should close #2198 . I removed my entry from built-in-functions and merged it into the existing iterators/iter entry.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/iterators/terms/iter/iter.md [hunk 1]
```diff
@@ -60,26 +57,45 @@ The following example applies the optional `sentinel` parameter through a callab
 
 ```py
 class codeNinjaCount:
```
Comment (SSwiniarski) [inline content/python/concepts/iterators/terms/iter/iter.md:59]: The example as it was worked, but I think we'd like to replace it with a completely new example, as codeNinja refers to another site.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nikolaspoczekaj so far, so good. My one comment is to replace the sentinel example completely as it contains references to another site.

**Comment by nikolaspoczekaj** [comment]
@SSwiniarski Done. I changed the example.

**Review by SSwiniarski — APPROVED** [review]
@nikolaspoczekaj looks good for review two.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks @nikolaspoczekaj! The edits should appear here: https://www.codecademy.com/resources/docs/python/iterators/iter

cc: @SSwiniarski @KTom101

**Pull request merged**